bin/rule-generator.js

```diff
@@ -0,0 +1,448 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const figlet = require('figlet');
+const inquirer = require('inquirer');
+const cmd = require('node-command-line');
+const axeFile = path.join(__dirname, '..', 'axe.js');
+const filehound = require('filehound');
+
+const DIRECTORIES = {
+	RULES: path.join(__dirname, '..', '/lib/rules'),
+	TEST_RULES_MATCHES: path.join(__dirname, '..', '/test/rule-matches'),
+	CHECKS: path.join(__dirname, '..', '/lib/checks'),
+	TEST_CHECKS_UNIT: path.join(__dirname, '..', '/test/checks'),
+	TEST_CHECKS_INTEGRATION: path.join(__dirname, '..', '/test/integration/rules')
+};
+
+const showBanner = () => {
+	const COLOR_DQ_HEX = '#0077c8';
+	console.log(chalk.hex(COLOR_DQ_HEX)(figlet.textSync('Axe Rule Generator')));
+};
+
+const QUESTIONS = {
+	GET_RULE_NAME: {
+		/**
+		 * What is the name of the RULE? (Eg: aria-valid):
+		 * 	- check if there is conflict in rule (file name)
+		 * 	- also if there is conflict in rule id itself
+		 */
+		name: 'GET_RULE_NAME',
+		type: 'input',
+		message: 'What is the name of the RULE? (Eg: aria-valid):',
+		validate: input => {
+			const ruleName = input.toLowerCase();
+			// 1) check if valid rule name
+			if (
+				!(
+					ruleName &&
+					ruleName.length > 0 &&
+					ruleName.search(/^[a-zA-Z-]+$/) !== -1
+				)
+			) {
+				throw new Error(
+					'RULE name should not be empty and can contain alphabets and dashes.'
```

Comment:
We should add the word "only" here, as in "can only contain alphabets and dashes."